arbitrum-docs/how-arbitrum-works/timeboost/timeboost-faq.mdx

```diff
@@ -0,0 +1,183 @@
+
+### Does it mean we expect searchers to bid continuously in advance, expecting opportunities to happen 1 minute later, rather than "in real time" opportunities (I see something → I submit an arbitrage tx with priority)?
+
+Before answering this question, it is worth clarifying that the participant will likely attempt to predict the amount of MEV generated between 15s and 1min 15s in the future - not 1 minute later. This prediction is because the auction is closed and resolved at a maximum of 15 seconds before the start of the next round (in the current proposal).
+
+Participants will bid continuously for the right to use the express lane in advance so that they (the participant) can profit from both (1) MEV opportunities they predict between 15s and 1min 15s in the future and (2) MEV opportunities in real-time during the period that the participant is in control of the express lane that they didn't otherwise predict in advance (proposed duration: 1 minute). If the participant does not win control of the express lane, opportunities that they see in real-time are still exploitable, but with a 200ms delay like all other txs (since only the express lane controller's transactions get sequenced with no delay).
+
+### When it comes to the implementation and integration, are there any references to Fastlane?
+
+We acknowledge FastLane's work and the perceived similarity in approach. We believe the design of Timeboost is fundamentally different from FastLane's design. We further believe that both designs aim to solve different problems. We understand that FastLane sold advantage separately for each AMM market and in peer-to-peer transaction propagation. Timeboost, on the other hand, sells an advantage in transaction inclusion timing and, therefore, sells the opportunity to profit from MEV altogether.
+
+Delphi Research, a member of the Arbitrum Research and Development Committee (ARDC), has published a great article comparing the transaction ordering policies of various solutions on child chains, including Timeboost, Optimism's Priority Gas Auctions (PGA), Polygon's FastLane, and OEV Network. Here is the [link](https://forum.arbitrum.foundation/t/transaction-ordering-policies-value-accrual-in-l2s-timeboost-op-pga-fastlane-oev-network-ardc-research-deliverables/26771) if you're curious to learn more!
+
+### Will the changes [the special rights being asked to be granted to the current sequencer operator] be made without community discussion or through on-chain voting?
+
+This AIP is proposing that the current sequencer operator has the autonomy to make the following changes only in circumstances where doing so would enhance Timeboost's long-term stability, preserve or improve the user experience for those using Timeboost-enabled Arbitrum chains, increase the security posture, resiliency, or stability of the chain, and/or otherwise help increase revenue for the ArbitrumDAO.
```

Comment:
```suggestion
This AIP is proposing that the current sequencer operator has the autonomy to make the following changes only in circumstances where doing so would enhance Timeboost's long-term stability, preserve or improve the user experience for those using Timeboost-enabled Arbitrum chains, increase security, resiliency, or stability of the chain, and/or otherwise help increase revenue for the ArbitrumDAO.
```